src/OrchardCore.Modules/OrchardCore.Html/Drivers/HtmlBodyPartDisplay.cs

```diff
@@ -40,7 +40,11 @@ public override IDisplayResult Edit(HtmlBodyPart HtmlBodyPart)
 
         public override async Task<IDisplayResult> UpdateAsync(HtmlBodyPart model, IUpdateModel updater)
```

Comment:
Then, maybe better to write here e.g `HtmlBodyPart part` in place of `HtmlBodyPart model`.